app/controllers/admin/content/system-images.js

```diff
@@ -0,0 +1,33 @@
+import Ember from 'ember';
+import { eventTopics } from 'open-event-frontend/utils/dictionary/event';
+import { keys, uniqBy } from 'lodash';
+
+const { Controller, computed } = Ember;
+
+export default Controller.extend({
+  imageTopic: keys(eventTopics)[0],
+
+  topics: computed(function() {
+    return keys(eventTopics);
+  }),
+
+  imageObjects: computed('imageTopic', function() {
```

Comment:
Also watch `model` in the computed property. This should react to any possible changes in the model too.